cmake/setup_dpdk_settings.dat.in

```diff
@@ -0,0 +1 @@
+RTE_SDK=@dpdk_INCLUDE_DIRS@
```

Comment:
Unfortunately this will not work 😕 
If you look at [`setup_dpdk.py`](https://github.com/seladb/PcapPlusPlus/blob/master/mk/setup_dpdk.py) we use `RTE_SDK` to load the [`KNI`](https://github.com/seladb/PcapPlusPlus/blob/22269e8d40bebdbef225d63bf0a92a9bd94988d7/mk/setup_dpdk.py#L834) or [`igb_uio`](https://github.com/seladb/PcapPlusPlus/blob/master/mk/setup_dpdk.py#L771) kernel modules, which are not available in the DPDK install directory (i.e `/usr/local/include`).

Since I want to keep things simple we can just change `setup_dpdk.py` to accept `RTE_SDK` path as a mandatory parameter [here](https://github.com/seladb/PcapPlusPlus/blob/22269e8d40bebdbef225d63bf0a92a9bd94988d7/mk/setup_dpdk.py#L1028).

That way the setup process will look something like this (assuming `-s` is the new parameter):
```shell
sudo python3 setup_dpdk.py setup -s /opt/seladb/dpdk-stable-20.11.3 -g 512 -i enp0s3 -m uio_pci_generic
```

We can also look for `RTE_SDK` in the environment variables and take if it exists... I can do this change in a separate PR after we merge this one.

So for now we only need to copy `setup_dpdk.py` and not create the `setup_dpdk_settings.dat`.

Reply:
Done in this commit: https://github.com/seladb/PcapPlusPlus/commit/6a864322ac1cbed0188ae5735597d28696ad3f94

I think we can remove this file altogether

Reply:
Thanks, I removed it and keep the copy and the script.